src/PackageDetailsViewExtension/PackageDetailsViewExtension.csproj

```diff
@@ -1,82 +1,82 @@
+	  <Name>DynamoCore</Name>
+	  <Private>False</Private>
+	</ProjectReference>
+	<ProjectReference Include="..\DynamoPackages\DynamoPackages.csproj">
+	  <Project>{47533B7C-0E1A-44A4-8511-B438645F052A}</Project>
+	  <Name>DynamoPackages</Name>
+	  <Private>False</Private>
+	</ProjectReference>
+	<ProjectReference Include="..\Libraries\PythonNodeModels\PythonNodeModels.csproj">
+	  <Project>{8872CA17-C10D-43B9-8393-5C5A57065EB0}</Project>
+	  <Name>PythonNodeModels</Name>
+	  <Private>False</Private>
+	</ProjectReference>
+	<ProjectReference Include="..\NodeServices\DynamoServices.csproj">
+	  <Project>{ef879a10-041d-4c68-83e7-3192685f1bae}</Project>
+	  <Name>DynamoServices</Name>
+	  <Private>False</Private>
+	</ProjectReference>
   </ItemGroup>
   <ItemGroup>
-    <Page Include="PackageDetailsView.xaml">
-      <SubType>Designer</SubType>
-      <Generator>MSBuild:Compile</Generator>
-    </Page>
+	<Page Include="PackageDetailsView.xaml">
+	  <SubType>Designer</SubType>
+	  <Generator>MSBuild:Compile</Generator>
+	</Page>
   </ItemGroup>
   <ItemGroup>
-    <Compile Update="Properties\Resources.Designer.cs">
-      <AutoGen>True</AutoGen>
-      <DesignTime>True</DesignTime>
-      <DependentUpon>Resources.resx</DependentUpon>
-    </Compile>
-    <Compile Update="Properties\Resources.en-US.Designer.cs">
-      <AutoGen>True</AutoGen>
-      <DesignTime>True</DesignTime>
-      <DependentUpon>Resources.en-US.resx</DependentUpon>
-    </Compile>
+	<Compile Update="Properties\Resources.Designer.cs">
+	  <AutoGen>True</AutoGen>
+	  <DesignTime>True</DesignTime>
+	  <DependentUpon>Resources.resx</DependentUpon>
+	</Compile>
+	<Compile Update="Properties\Resources.en-US.Designer.cs">
+	  <AutoGen>True</AutoGen>
+	  <DesignTime>True</DesignTime>
+	  <DependentUpon>Resources.en-US.resx</DependentUpon>
+	</Compile>
   </ItemGroup>
   <ItemGroup>
-    <EmbeddedResource Update="Properties\Resources.resx">
-      <Generator>ResXFileCodeGenerator</Generator>
-      <LastGenOutput>Resources.Designer.cs</LastGenOutput>
-    </EmbeddedResource>
-    <EmbeddedResource Update="Properties\Resources.en-US.resx">
-      <Generator>ResXFileCodeGenerator</Generator>
-      <LastGenOutput>Resources.en-US.Designer.cs</LastGenOutput>
-    </EmbeddedResource>
+	<EmbeddedResource Update="Properties\Resources.resx">
+	  <Generator>ResXFileCodeGenerator</Generator>
+	  <LastGenOutput>Resources.Designer.cs</LastGenOutput>
+	</EmbeddedResource>
+	<EmbeddedResource Update="Properties\Resources.en-US.resx">
+	  <Generator>ResXFileCodeGenerator</Generator>
+	  <LastGenOutput>Resources.en-US.Designer.cs</LastGenOutput>
+	</EmbeddedResource>
   </ItemGroup>
   <Target Name="AfterBuildOps" AfterTargets="Build">
-    <ItemGroup>
-      <ExtensionDefinition Include="PackageDetails_ViewExtensionDefinition.xml" />
-    </ItemGroup>
-    <Copy SourceFiles="@(ExtensionDefinition)" DestinationFolder="$(OutputPath)\viewExtensions\" />
+	<ItemGroup>
```

Comment:
please revert all this whitespace changes.

Reply:
Done